config/locales/en.yml

```diff
@@ -17,3 +17,9 @@ en:
     permission_priority: "Priority"
     permission_visible: "Visible"
     permission_boolean: "Boolean"
```

Comment:
@nimish-mehta On the permissions listing page admin/permissions and the new/edit form, can we add hint text for user. currently not clear what Boolean mean here.